tests/ui/conftest.py

```diff
@@ -14,31 +24,103 @@ def capabilities(capabilities):
     capabilities['marionette'] = True
     return capabilities
 
+
 @pytest.fixture
 def fxa_account(base_url):
-    url = DEV_URL if 'dev' in base_url else PROD_URL
+    url = DEV_URL if 'dev' or 'localhost' in base_url else PROD_URL
     return FxATestAccount(url)
 
 
-@pytest.fixture(scope='session')
-def jwt_issuer(base_url, variables):
+@pytest.fixture
+def jwt_issuer(base_url, json_file):
     try:
-        hostname = [urlparse.urlsplit(base_url).hostname]
-        return variables['api'][hostname]['jwt_issuer']
+        hostname = urlparse.urlsplit(base_url).hostname
+        return json_file['api'][hostname]['jwt_issuer']
     except KeyError:
         return os.getenv('JWT_ISSUER')
 
 
-@pytest.fixture(scope='session')
-def jwt_secret(base_url, variables):
+@pytest.fixture
+def jwt_secret(base_url, json_file):
     try:
-        hostname = [urlparse.urlsplit(base_url).hostname]
-        return variables['api'][hostname]['jwt_secret']
+        hostname = urlparse.urlsplit(base_url).hostname
+        return json_file['api'][hostname]['jwt_secret']
     except KeyError:
         return os.getenv('JWT_SECRET')
 
 
 @pytest.fixture
+def initial_data(transactional_db, live_server, base_url):
+    call_command('generate_addons', 10, app='firefox')
+
+
+@pytest.fixture
+def create_superuser(transactional_db, live_server, base_url):
+    hostname = urlparse.urlsplit(base_url).hostname
+    create_switch('super-create-accounts')
+    call_command('loaddata', 'initial.json')
+
+    call_command(
+        'createsuperuser',
+        interactive=False,
+        username='uitest',
+        email='uitester@mozilla.org',
+        add_to_supercreate_group=True,
+        save_api_credentials='tests/ui/variables.json',
+        hostname=hostname
+    )
+
+
+@pytest.fixture
+def force_user_login():
+    from olympia.users.models import UserProfile
+    user = UserProfile.objects.get(username='uitest')
+    return user
+
+
+@pytest.fixture
+def user(
+        transactional_db, create_superuser, our_live_server, base_url,
+        fxa_account, jwt_token):
+    url = '{base_url}/api/v3/accounts/super-create/'.format(base_url=base_url)
+
+    params = {
+        'email': fxa_account.email,
+        'password': fxa_account.password,
+        'username': fxa_account.email.split('@')[0],
+        'fxa_id': fxa_account.session.uid}
+    print(fxa_account.password)
+    headers = {'Authorization': 'JWT {token}'.format(token=jwt_token)}
+    response = requests.post(url, data=params, headers=headers)
+    user = {
+        'email': fxa_account.email,
+        'password': fxa_account.password,
+        'username': fxa_account.email.split('@')[0],
+        'fxa_id': fxa_account.session.uid
+    }
+    assert requests.codes.created == response.status_code
+    user.update(response.json())
+    print(user)
+    return user
+
+
+@pytest.fixture(scope='function')
+def our_live_server(request):
```

Comment:
Can you add comments to help explain why we're implementing our own liveserver fixture. Also, we shouldn't need the `our_` prefix as we can just replace the existing fixture of the same name.